### DISCUSSION TITLE
Linking Files for creating a map in directory/environment

### DISCUSSION THREAD

**Discussion by nathansturgill** [giswqs/geohey#10] [Q&A] [answered]
Hi everyone, I was curious if anyone else ran into this problem when trying to link the files to the Python package you created in your environment to then install the package to test it. 
![image](https://github.com/giswqs/geohey/assets/147578367/46028700-068e-4fc7-8aaa-a6cd5d881333)

**Comment by giswqs** [comment, marked as answer]
Are you sure you are under the root directory of the repo? Your file path does not look right. 

```bash
git clone https://github.com/nathansturgill/sturgmap.git
cd sturgmap
pip install -e .
```

**Reply by nathansturgill** [reply]
Hello Dr. Wu,

I recently fixed this issue and I got it under the correct root directory of the repo. 
I now have another issue I will email you about.
